### DISCUSSION TITLE
Welcome to organization Discussions!

### DISCUSSION THREAD

**Discussion by scott-wilson** [vfx-rs/organization#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by davvid** [comment]
Thanks for opening this up, @scott-wilson! Looking forward to the discussions.

I'm the author of http://github.com/git-cola/git-cola/ so I'm familiar with the challenges of maintaining cross-platform Python tools and applications. I'm a long-time C++ and Python developer and Rust solves some very real problems in the domains we typically use these tools in so I'm excited for the future of what we can build together.

I've been working on https://github.com/vfx-rs/ptex-bind lately as a great way to get deeper into Rust in a way that leverages the expertise I already have in the C++ ecosystem. cppmm made the sys bindings so easy to make that I'm already well along on the high-level ptex-rs Rust API. Thanks again @anderslanglands !
